analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -191,6 +193,15 @@ def individual_gpc(individual_ascii, validation_table, log=None):
     This function is currently repetitive of the function above for the individual detection cases.
     However, I am going to keep it here in the case that we want to plot the individual detections against
     the binned detections.
```

Comment:
Consider limited it to 79 characters wide by wrapping the text

Reply:
I thought PyCharm was set to 79, but it turns out it was set to 120. Went back through code and fixed this. commit 359a259